packages/widget-v2/src/pages/SwapPage/SwapDetailModal.tsx

```diff
@@ -31,7 +31,7 @@ export const SwapDetailModal = createModal((modalProps: ModalProps) => {
   const clientOperations = route && getClientOperations(route.operations);
 
   const usesEvmInOperations = useMemo(() => {
-    return clientOperations?.find(operation => operation.toChainID === "1");
+    return clientOperations?.find((operation) => operation.toChainID === "1");
```

Comment:
this logic is wrong, we need to check `toChainID`'s `chainType`

Reply:
@codingki we were missing this disclaimer in SwapExecutionPageRouteDetailed so I made a component with the new logic inside

![Screenshot 2024-10-29 at 12 13 39 PM](https://github.com/user-attachments/assets/debf08b4-0985-4f90-92f6-9583d87ba1f4)